### DISCUSSION TITLE
debug using a flag like `--debug`

### DISCUSSION THREAD

**Discussion by TaylorHo** [oclif/oclif#1274] [Q&A] [answered]
Hello! I would like to know, is it possible to use the tool debug through a flag, such as `--debug`, without having to configure a `DEBUG=*` variable for this?

**Comment by mdonnalley** [comment, marked as answer]
@TaylorHo Yes there's a few ways to do this.

Here's what we do for the Salesforce CLI: https://github.com/salesforcecli/cli/blob/main/src/flags.ts

Also, there are a couple of solutions posted in #143 that should work for you

**Reply by TaylorHo** [reply]
Using it like in the Salesforce CLI worked perfectly! Thank you!
